web/src/app/header/titlebar.component.sass

```diff
@@ -44,7 +44,7 @@ $menu-width: 135px
     display: flex
     justify-content: center
     align-items: center
-    background-color: mat.m2-get-color-from-palette(mat.$yellow-palette,900)
+    background-color: mat.m2-get-color-from-palette(mat.$m2-yellow-palette,900)
```

Comment:
nit: Unintentionally staged?

Reply:
No, it's intended.

I disabled the feature to display viewer mode status on the PR #42, and the style blocks are not used from that.
Then I noticed I needed to upgrade the property name for newer Angular material.